Sources/Core/BloctoSDK.swift

```diff
@@ -130,6 +130,13 @@ public class BloctoSDK {
     public func application(
         open url: URL
     ) {
+        if let scheme = url.scheme,
+           scheme.lowercased().hasPrefix("blocto") {
```

Comment:
I'd prefer to use specific schemes. It's better to understand.
```suggestion
        if let scheme = url.scheme?.lowercased(),
           scheme == "blocto" || scheme == "blocto-staging" {
```